### PR TITLE
fix: webcache 404 error page

### PR DESCRIPTION
**Pull request by baby230211**
## Types of changes

* **Bugfix**

## Description

fix 404 error page on google cache page

## Steps to Test This Pull Request

open in google link site:https://tw.pycon.org
choose in search result "頁庫存擋"
you open page https://webcache.googleusercontent.com/search?q=cache:9er0RXJ1v60J:https://tw.pycon.org/&cd=1&hl=zh-TW&ct=clnk&gl=tw&cd=1&hl=ru&ct=clnk&gl=ua
then page will redirect to https://tw.pycon.org/2023/en-us/search?q=cache%3A9er0RXJ1v60J%3Ahttps%3A%2F%2Ftw.pycon.org%2F&cd=1&hl=zh-TW&ct=clnk&gl=tw

## Expected behavior

show correctly on webcache page

## Related Issue

refer to https://github.com/vuejs/vue-router/issues/2042

### CODE REVIEW

**Comment by mattwang44** [comment]
Would you please elaborate how does the error occur and what's the approach to solve it?

**Comment by mattwang44** [comment]
thanks for editing!

**Comment by mattwang44** [comment]
@baby230211 Thanks for the effort. 

I don't know web cache tech really well, but I still have concerns about introducing `@nuxtjs/router` (which seems to be out of maintenance for a while) and handling the logic on our own. I'd prefer [upgrading to Nuxt3 if it really does help](https://github.com/vuejs/vue-router/issues/2042#issuecomment-1426804443).

Please launch a discussion with teammates on Discord:
- How urgent is this issue? As we'll keep maintaining the pycontw websites across different years, do people really need the web cache of it?
- Which solution requires the least maintenance effort?
- I found out the same issues happened on the 2021 & 2022 websites (though they've been archived). We probably need to survey how to solve the issue for the archived static websites.

**Review by yungshenglu — APPROVED** [review]

File: nuxt.config.js [hunk 1]
```diff
@@ -79,6 +79,14 @@ export default {
         '@nuxtjs/tailwindcss',
         // https://github.com/nuxt-community/dotenv-module
         '@nuxtjs/dotenv',
+        [
+            // 1
```
Comment (mattwang44) [inline nuxt.config.js:83]: 這裡的註解與 configs/router.js 裡的編號註解是否要刪掉？還是他有其他意義？

**Review by mattwang44 — APPROVED** [review]
LGTM
麻煩在幫忙看一下 2021 & 2022 網站的相同問題是否要另外開 issue 紀錄 🙏🏽

File: nuxt.config.js (hunk 1 above)
Comment (baby230211) [reply nuxt.config.js]: updated 我另外開票紀錄2021 & 2022

**Review by mattwang44 — APPROVED** [review]
LGTM

**Comment by josix** [comment]
Thank you all for working on this

**Pull request merged**